vcpkg/consuming-packages/configure-git-authentication-in-github-actions.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Configure Git authentication in GitHub Actions
+description: Learn to configure Git credentials for private repositories when using vcpkg on GitHub Actions
+author: vicroms
+ms.author: viromer
+ms.date: 7/20/2023
+ms.prod: vcpkg
+---
+# Configure Git authentication in GitHub Actions
+
+When using vcpkg with private registries it is necessary to provide access
+credentials to authenticate Git operations executed internally by vcpkg.
+
+In this article we will show you how to authenticate to a private vcpkg registry
+hosted on GitHub to use in your GitHub Actions workflows.
+
+## Step 1: Create a Personal Access Token for your private repository
+
+The first step is to create a Personal Access Token (PAT) for your GitHub account.
+
+- Log into GitHub using an account with access to the private repository.
+- Go to <https://github.com/settings>
+- Click on _Developer Settings_
+- Click on _Tokens (classic)_
+- Click the _Generate new token (classic)_
+- Give your token appropiate description, expiration date, and permissions. Make
+  sure that the `repo` permission is enabled, this will give the PAT full access
+  to your private repositories.
+- Click on _Generate token_
+- Copy and paste the generated PAT in a secure location, you won't be able to
+  view it again after leaving the current page
+- (Optional) If your organization uses SSO, you need to enable it for the PAT
+
+### Step 2: Create a secret to use in your workflow
+
+> !NOTE
+> Never store a GitHub Personal Access Token in your repository files
```

Comment:
Maybe this NOTE should move to the above step?

Reply:
Great suggestion